unlock-app/src/pages/log.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from 'react'
+import { connect } from 'react-redux'
+import Head from 'next/head'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import UnlockPropTypes from '../propTypes'
+import Layout from '../components/interface/Layout'
+import CreatorAccount from '../components/creator/CreatorAccount'
+import BrowserOnly from '../components/helpers/BrowserOnly'
+import GlobalErrorConsumer from '../components/interface/GlobalErrorConsumer'
+import GlobalErrorProvider from '../utils/GlobalErrorProvider'
+import { pageTitle } from '../constants'
+import Web3Service from '../services/web3Service'
+
+export const Log = ({ account, network, lockAddresses }) => {
+  const [transactions, setTransactions] = useState([])
+  const w3s = new Web3Service()
```

Comment:
Huh... I do not think we should access `Web3Service` from here especially since all transactions are already in the redux store?